src/overlords/mining/miner.ts

```diff
@@ -276,12 +340,13 @@ export class MiningOverlord extends Overlord {
 	 */
 	private linkMiningActions(miner: Zerg) {
 
-		// Approach mining site
-		if (this.goToMiningSite(miner)) return;
-
 		// Link mining
 		if (this.link) {
-			miner.harvest(this.source!);
+			const res = miner.harvest(this.source!);
```

Comment:
this seems to do the same as what it was before - if this is true, change it back to keep consistency

Reply:
Think about ordering. Link miners spend 95%+ of their life stationary mining. They move less than 20 ticks in the beginning. Far more expensive to check every tick if they should move rather than trying to mine every tick and moving them if they can't.